EventsExpress.Db/Entities/RefreshToken.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace EventsExpress.Db.Entities
+{
+  public  class RefreshToken
+    {
+        public int Id { get; set; }
```

Comment:
inherit BaseEntity to have Id automatically

Reply:
yes, but I think is changing nothing if I write this manually